ts/Series/Treegraph/TreegraphSeries.ts

```diff
@@ -342,86 +378,120 @@ class TreegraphSeries extends OrganizationSeries {
         return { ax, bx, ay, by };
     }
 
+    getLinks(): TreegraphLink[] {
+        const series = this;
+        const links = [] as TreegraphLink[];
+        this.data.forEach((point): void => {
+            if (point.node.parent) {
+                const link = new series.LinkClass().init(
```

Comment:
Aren't we able to call `linkClass init` with all the below declarations? like `link.toNode`, `link.formatPrefix` etc.

Reply:
+ hardcoded props can be defined in constructor:
```
                link.formatPrefix = 'link';
                link.dataLabelOnNull = true;
```